api-gateway/src/main/resources/application.yml

```diff
@@ -0,0 +1,129 @@
+# This file contains shared properties across all environments; it is always loaded by Spring
+# See https://github.com/department-of-veterans-affairs/abd-vro/wiki/Configuration-settings#vros-use-of-spring-profiles
+
+management:
+  endpoints.web:
+    exposure.include: "*"
+  endpoint:
+    health:
+      show-details: always
+      probes:
+        enabled: true
+      group:
+        liveness.include: livenessState
+        readiness.include: readinessState, db
+  metrics:
+    enabled: true
+    distribution:
+      percentiles.http.server.requests: 0.5, 0.90, 0.95, 0.99, 0.999
+      percentiles-histogram.http.server.requests: true
+      sla.http.server.requests: 10ms, 50ms
+      slo.http.server.requests: 10ms, 50ms
+    tags:
+      group: starter
+      service: example
+      region: "${POD_REGION:local}"
+      stack: "${CLUSTER:dev}"
+      ns: "${NAMESPACE:example}"
+      pod: "${POD_ID:docker}"
+    web.server.request.autotime.enabled: true
+  server.port: 8061
+
+server:
+  ssl:
+    # TODO: enable SSL
+    enabled: false
+  port: 8060
+  maxHttpHeaderSize: 48000
+  session:
+    timeout: 60
+  connection:
+    timeout: 60000
+  servlet:
+    session:
+      timeout: 120000
+
+spring:
+  application:
+    name: "vro-api"
+  servlet:
+    multipart:
+      maxFileSize: 25MB
+      maxRequestSize: 25MB
+      enabled: true
+    session:
+      timeout: 120000
+  resources:
+    add-mappings: false
+  http:
+    encoding:
+      force: true
+
+springdoc:
+  writer-with-default-pretty-printer: true
+  show-actuator: true
+  swagger-ui:
+    # http://localhost:8078/swaggerui will be forwarded to http://localhost:8078/webjars/swagger-ui/index.html
+    path: /swaggerui
+    operations-sorter: method
+    tagsSorter: alpha
+    # Populate API dropdown on the upper-right of Swagger UI
+    urls:
+      - name: 0. Gateway API
+        # API defined for this API Gateway
+        url: /v3/api-docs
+      - name: App API
+        # API defined for the VRO Java-base App
+        url: /app/v3/api-docs
+      - name: Contention Classification API
+        # Use the route defined under spring.cloud.gateway below
+        url: /contention-classification/openapi.json
```

Comment:
@lukey-luke If you the location of the `openapi.json` changes, please update this line.